CreateRestaurantFunction.cs

```diff
@@ -18,28 +18,29 @@ public static class CreateRestaurantFunction
         {
             log.Info("CreateRestaurant function processed a request.");
 
-            string strMongoDBAtlasUri = System.Environment.GetEnvironmentVariable("MongoDBAtlasURI");
-            log.Info($"Atlas connection string is { strMongoDBAtlasUri}");
+            var strMongoDBAtlasUri = System.Environment.GetEnvironmentVariable("MongoDBAtlasURI");
+            log.Info($"Atlas connection string is {strMongoDBAtlasUri}");
 
-            MongoUrl mongoUrl = new MongoUrl(strMongoDBAtlasUri);
+            var mongoUrl = new MongoUrl(strMongoDBAtlasUri);
             var settings = MongoClientSettings.FromUrl(mongoUrl);
             //for more on why we're using ServerSelectionTimeout, read https://scalegrid.io/blog/understanding-mongodb-client-timeout-options/
-            settings.ServerSelectionTimeout = new System.TimeSpan(0, 0, 5);
+            settings.ServerSelectionTimeout = TimeSpan.FromSeconds(5);
 
             var client = new MongoClient(settings);           
```

Comment:
I confirmed this (having run into that issue with AWS Lambda) and further connections (after the first one) are indeed much faster so it looks like it does reuse the same container/VM and the driver can reuse the same connection for further calls. good suggestion about a singleton to hammer home that we don't want to be recreating this everytime, I'll probably do another pass at it.